### PR TITLE
Feat: Added Cart Page.

### PR DESCRIPTION
**Pull request by coldpigli**
Added the cart page.

Desktop View:
image

Mobile View:
image
image

### CODE REVIEW

**Review by raman1999 — APPROVED** [review]
well written code , everything looks great to me

**Comment by Gautam2305** [comment]
code looks clean, great that you added responsiveness with media queries. Looks good to merge

File: routes/cart-page.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+                        <a href="#"><span class="add-to-bag material-icons md-24">delete</span></a>
+                    </div>
+                </div>
+                <div class="cart-item w-100 flex children-centered">
+                    <div class="product-img">
+                        <img src="../assets/patato.png" alt="food">
+                    </div>
+                    <div class="product-decription">
+                        <h3 class="heading3">Chilli</h3>
+                        <p class="paragraph2 txt-gray">Bohot teekhi, laal laal</p>
```
Comment (vaibhav-mahalle) [inline routes/cart-page.html:85]: nice text heading.

File: routes/cart-page.html [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+                    </div>
+               </div>
+            </div>
+        </div>
+    </section>
+    <footer class="footer w-75 flex-vertical children-middle">
+        <h1 class="heading2">Download Our App</h1>
+        <div class="footer-download flex">
+            <div class="download-img">
+                <img src="../assets/googleplay-btn.png" alt="">
```
Comment (vaibhav-mahalle) [inline routes/cart-page.html]: add alt description here.

**Review by vaibhav-mahalle — APPROVED** [review]
added some suggestions please look into it.

File: routes/cart-page.html (hunk 2 above)
Comment (coldpigli) [reply routes/cart-page.html]: will do. thanks

**Pull request merged**